src/test/java/org/folio/rest/api/InstanceStaffSuppressMigrationScriptTest.java

```diff
@@ -0,0 +1,25 @@
+package org.folio.rest.api;
+
+import org.folio.rest.jaxrs.model.Instance;
+
+public class InstanceStaffSuppressMigrationScriptTest
```

Comment:
Would it be better to have parameterized tests instead of building hierarchy of classes?